package.json

```diff
@@ -63,10 +63,13 @@
     "webpack-dev-server": "^2.9.3"
   },
   "scripts": {
-    "lint": "standard \"./**/*.js\" | snazzy",
-    "lint:fix": "standard \"./**/*.js\" --fix | snazzy",
     "unit": "jest --testPathPattern=.*.test.js$",
-    "e2e":  "jest --testPathPattern=.*e2e.*"
+    "e2e":  "jest --testPathPattern=.*e2e.* --runInBand",
```

Comment:
@akegan I'm leaving `unit` and `e2e` commands, since this is the way it's setup in semaphore and we can have this PR passing. 
We could later create  chore for removing this and merge in all the open PRs. if we change the setup in semaphore to `test:unit` and `test:e2e` the other PRs will fail. 
Or we can just let the other PRs fail until we merge this one and let the other PR pull from master.

Reply:
yeah I think it's fine to have the other PRs fail until this is merged if it's ready to go.